src/puppetlabs/puppetdb/cli/import.clj

```diff
@@ -102,11 +107,23 @@
            construct-base-url
            utils/validate-cli-base-url!)))))
 
+(defn import!
+  [infile command-versions command-fn]
+  (with-open [tar-reader (archive/tarball-reader infile)]
+    (doseq [tar-entry (archive/all-entries tar-reader)]
+      (process-tar-entry command-fn tar-reader tar-entry command-versions))))
+
 (defn -main
   [& args]
   (let [{:keys [infile base-url]} (validate-cli! args)
-        command-versions (:command_versions (parse-metadata infile))]
-    ;; TODO: do we need to deal with SSL or can we assume this only works over a plaintext port?
-    (with-open [tar-reader (archive/tarball-reader infile)]
-      (doseq [tar-entry (archive/all-entries tar-reader)]
-        (process-tar-entry tar-reader tar-entry base-url command-versions)))))
+        import-archive (fs/normalized-path infile)
```

Comment:
It's a poorly named function, whose name was actually changed in the most recent version of the library